lib/models.ts

```diff
@@ -90,3 +90,33 @@ export interface ParsedOutput {
   // Only valid when streaming is complete. Don't use while streaming
   completed: boolean;
 }
+
+export interface OpenAPISchema {
+  type?: "array" | "object"; // More are probably valid
+  properties?: { [key: string]: any };
+  required?: string[];
+  items?: {
+    properties?: { [key: string]: any };
+    required?: string[];
+  };
+}
```

Comment:
What's wrong with using the types from here? They should be more rigorously defined
```
import { OpenAPIV3_1 } from "openapi-types";
```